### ISSUE TITLE
can you get the api to return JSON?

### ISSUE THREAD

**Issue by wbrenton** [runpod/runpodctl#148]
Seems like some core functionality is missing, if its not its poorly documented.

Is it possible to get runpodctl to return json?

**Comment by jojje** [comment]
The output is unfortunately not centralized. It's spread all over the various sub-commands.
I'd have to refactor much of the CLI to enable this, and tend to refrain from such on repos I don't own. So, is there some specific subcommand you're interested in getting json output for? `get pod` I'd assume. Anything else that's critical?

**Comment by jojje** [comment]
@romosa I added a `-j` / `--json` flag to `get pod`.
Check the [sample output](https://github.com/runpod/runpodctl/pull/222) and let me know if this works for you.

**Comment by romosa** [comment]
Yes, it works. We were able to easily pickup TCP Connection details now and no longer relying on regex. Thank you @jojje
